.ci/atime/tests.R

```diff
@@ -108,7 +108,7 @@ test.list <- atime::atime_test_list(
 
   # Issue reported in: https://github.com/Rdatatable/data.table/issues/5426
   # To be fixed in: https://github.com/Rdatatable/data.table/pull/5427
-  "setDT improved in #5427" = atime::atime_test(
+  "setDT improved fixed in #5427" = atime::atime_test(
```

Comment:
please undo this change as it is unrelated to the test you are adding in this PR